src/routes/home/index.js

```diff
@@ -25,7 +26,12 @@ async function action({ fetch }) {
     chunks: ['home'],
     component: (
       <Layout>
-        <Home typeofcomplaintValues={typeofcomplaintValues} />
+        <Home
+          typeofcomplaintValues={typeofcomplaintValues}
+          boroughBoundariesFeatureCollection={
+            boroughBoundariesFeatureCollection
+          }
```

Comment:
One concern I have with sending the GeoJSON down as part of the initial page load like this is that if it's bundled into the same HTTP request as the UI, then any updates will bust the cache as usual, but an additional 1MB or so will need to be loaded, based on the gzipped size of the file:

```
$ cat public/borough-boundaries-clipped-to-shoreline.geo.json  | gzip -9 | wc -c
 1036075
```

That said, this is an app designed for uploading multiple multi-megabyte photos, so maybe the occasional extra 1 megabyte download isn't so bad.